section2/exercises/if-statements.js

```diff
@@ -82,14 +90,22 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+var cupsOfFlour = 3;
 var hasSauce = true;
 
 // Write your conditional here
-
+if (cupsOfFlour < 2 && hasSauce === true) {
+  console.log("I cannot make pizza");
+} else if (cupsOfFlour < 2 && hasSauce === false) {
+  console.log("I cannot make pizza");
+} else if (cupsOfFlour >= 2 && hasSauce === true) {
+  console.log("I can make pizza");
+} else if (cupsOfFlour >= 2 && hasSauce === false) {
+  console.log("I cannot make pizza");
+}
 
 /*
-For the last exercise, an ideal solution probably uses a logical operator. 
+For the last exercise, an ideal solution probably uses a logical operator.
 Did yours? Do you know what a logical operator in JavaScript is? Google it to answer
 for yourself!
 
```

Comment:
Great style and syntax throughout (the use white space, spaces, and indentation)